packages/origin-backend/src/pods/certificate/certification-request-queue-item.entity.ts

```diff
@@ -0,0 +1,29 @@
+import { Entity, Column, Unique, PrimaryGeneratedColumn } from 'typeorm';
+import { Min } from 'class-validator';
+import { ExtendedBaseEntity } from '../ExtendedBaseEntity';
+import { CertificationRequestQueueItemDTO } from './certification-request-queue-item.dto';
+
+@Entity()
+@Unique(['deviceId', 'fromTime', 'toTime'])
+export class CertificationRequestQueueItem extends ExtendedBaseEntity
+    implements CertificationRequestQueueItemDTO {
+    @PrimaryGeneratedColumn()
+    id: number;
+
+    @Column('varchar', { nullable: true })
```

Comment:
This is conflicting with 

```
@Validate(PositiveBNStringValidator)
    public readonly energy: string;
```

in DTO

Reply:
You're correct, this shouldn't be nullable.